{
  "uid" : "ab6877f419c6b501",
  "name" : "Verify that the email link calls an email client",
  "fullName" : "tests.contacts_page_test.TestContactsPageLinks#test_cp_03_03_verify_email_link_calls_an_email_client",
  "historyId" : "439979364f81ff50b7b18242ebace4b6",
  "time" : {
    "start" : 1728694530009,
    "stop" : 1728694530009,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x5598b2f7f02a <unknown>\n#1 0x5598b2c655e0 <unknown>\n#2 0x5598b2c5d7eb <unknown>\n#3 0x5598b2c4da09 <unknown>\n#4 0x5598b2c4f744 <unknown>\n#5 0x5598b2c4dcbd <unknown>\n#6 0x5598b2c4d52f <unknown>\n#7 0x5598b2c4d422 <unknown>\n#8 0x5598b2c4b493 <unknown>\n#9 0x5598b2c4bada <unknown>\n#10 0x5598b2c67f41 <unknown>\n#11 0x5598b2cf98c5 <unknown>\n#12 0x5598b2cd9b22 <unknown>\n#13 0x5598b2cf8d7d <unknown>\n#14 0x5598b2cd98c3 <unknown>\n#15 0x5598b2ca76b3 <unknown>\n#16 0x5598b2ca868e <unknown>\n#17 0x5598b2f49a2b <unknown>\n#18 0x5598b2f4d9b1 <unknown>\n#19 0x5598b2f36225 <unknown>\n#20 0x5598b2f4e532 <unknown>\n#21 0x5598b2f1b38f <unknown>\n#22 0x5598b2f6df28 <unknown>\n#23 0x5598b2f6e0f3 <unknown>\n#24 0x5598b2f7de7c <unknown>\n#25 0x7f22c2894ac3 <unknown>",
  "statusTrace" : "driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"49c5a576d8197f85cf4dd395228b4693\")>\n\n    @pytest.fixture()\n    @allure.step(f'Open page: {MainPageLinks.URL_MAIN_PAGE}')\n    def main_page_open(driver):\n>       driver.get(MainPageLinks.URL_MAIN_PAGE)\n\ntests/conftest.py:22: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:353: in get\n    self.execute(Command.GET, {\"url\": url})\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:344: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7f8f5b51a420>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"unknown error: net::ERR_CONNECTION_REFUSED\\\\n  ...\\\\n#23 0x5598b2f6e0f3 \\\\u003Cunknown>\\\\n#24 0x5598b2f7de7c \\\\u003Cunknown>\\\\n#25 0x7f22c2894ac3 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\nE         (Session info: chrome=129.0.6668.89)\nE       Stacktrace:\nE       #0 0x5598b2f7f02a <unknown>\nE       #1 0x5598b2c655e0 <unknown>\nE       #2 0x5598b2c5d7eb <unknown>\nE       #3 0x5598b2c4da09 <unknown>\nE       #4 0x5598b2c4f744 <unknown>\nE       #5 0x5598b2c4dcbd <unknown>\nE       #6 0x5598b2c4d52f <unknown>\nE       #7 0x5598b2c4d422 <unknown>\nE       #8 0x5598b2c4b493 <unknown>\nE       #9 0x5598b2c4bada <unknown>\nE       #10 0x5598b2c67f41 <unknown>\nE       #11 0x5598b2cf98c5 <unknown>\nE       #12 0x5598b2cd9b22 <unknown>\nE       #13 0x5598b2cf8d7d <unknown>\nE       #14 0x5598b2cd98c3 <unknown>\nE       #15 0x5598b2ca76b3 <unknown>\nE       #16 0x5598b2ca868e <unknown>\nE       #17 0x5598b2f49a2b <unknown>\nE       #18 0x5598b2f4d9b1 <unknown>\nE       #19 0x5598b2f36225 <unknown>\nE       #20 0x5598b2f4e532 <unknown>\nE       #21 0x5598b2f1b38f <unknown>\nE       #22 0x5598b2f6df28 <unknown>\nE       #23 0x5598b2f6e0f3 <unknown>\nE       #24 0x5598b2f7de7c <unknown>\nE       #25 0x7f22c2894ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py:229: WebDriverException",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "clear_allure_results_folder",
    "time" : {
      "start" : 1728694482412,
      "stop" : 1728694482412,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "driver",
    "time" : {
      "start" : 1728694530009,
      "stop" : 1728694530776,
      "duration" : 767
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "main_page_open",
    "time" : {
      "start" : 1728694530776,
      "stop" : 1728694531412,
      "duration" : 636
    },
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x5598b2f7f02a <unknown>\n#1 0x5598b2c655e0 <unknown>\n#2 0x5598b2c5d7eb <unknown>\n#3 0x5598b2c4da09 <unknown>\n#4 0x5598b2c4f744 <unknown>\n#5 0x5598b2c4dcbd <unknown>\n#6 0x5598b2c4d52f <unknown>\n#7 0x5598b2c4d422 <unknown>\n#8 0x5598b2c4b493 <unknown>\n#9 0x5598b2c4bada <unknown>\n#10 0x5598b2c67f41 <unknown>\n#11 0x5598b2cf98c5 <unknown>\n#12 0x5598b2cd9b22 <unknown>\n#13 0x5598b2cf8d7d <unknown>\n#14 0x5598b2cd98c3 <unknown>\n#15 0x5598b2ca76b3 <unknown>\n#16 0x5598b2ca868e <unknown>\n#17 0x5598b2f49a2b <unknown>\n#18 0x5598b2f4d9b1 <unknown>\n#19 0x5598b2f36225 <unknown>\n#20 0x5598b2f4e532 <unknown>\n#21 0x5598b2f1b38f <unknown>\n#22 0x5598b2f6df28 <unknown>\n#23 0x5598b2f6e0f3 <unknown>\n#24 0x5598b2f7de7c <unknown>\n#25 0x7f22c2894ac3 <unknown>\n\n",
    "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
    "steps" : [ {
      "name" : "Open page: https://brainup.site/",
      "time" : {
        "start" : 1728694530776,
        "stop" : 1728694531411,
        "duration" : 635
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x5598b2f7f02a <unknown>\n#1 0x5598b2c655e0 <unknown>\n#2 0x5598b2c5d7eb <unknown>\n#3 0x5598b2c4da09 <unknown>\n#4 0x5598b2c4f744 <unknown>\n#5 0x5598b2c4dcbd <unknown>\n#6 0x5598b2c4d52f <unknown>\n#7 0x5598b2c4d422 <unknown>\n#8 0x5598b2c4b493 <unknown>\n#9 0x5598b2c4bada <unknown>\n#10 0x5598b2c67f41 <unknown>\n#11 0x5598b2cf98c5 <unknown>\n#12 0x5598b2cd9b22 <unknown>\n#13 0x5598b2cf8d7d <unknown>\n#14 0x5598b2cd98c3 <unknown>\n#15 0x5598b2ca76b3 <unknown>\n#16 0x5598b2ca868e <unknown>\n#17 0x5598b2f49a2b <unknown>\n#18 0x5598b2f4d9b1 <unknown>\n#19 0x5598b2f36225 <unknown>\n#20 0x5598b2f4e532 <unknown>\n#21 0x5598b2f1b38f <unknown>\n#22 0x5598b2f6df28 <unknown>\n#23 0x5598b2f6e0f3 <unknown>\n#24 0x5598b2f7de7c <unknown>\n#25 0x7f22c2894ac3 <unknown>\n\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "driver",
        "value" : "<selenium.webdriver.chrome.webdriver.WebDriver (session=\"49c5a576d8197f85cf4dd395228b4693\")>"
      } ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  } ],
  "afterStages" : [ {
    "name" : "driver::0",
    "time" : {
      "start" : 1728694531444,
      "stop" : 1728694531506,
      "duration" : 62
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "The Contacts Page"
  }, {
    "name" : "parentSuite",
    "value" : "tests"
  }, {
    "name" : "suite",
    "value" : "contacts_page_test"
  }, {
    "name" : "subSuite",
    "value" : "TestContactsPage"
  }, {
    "name" : "host",
    "value" : "fv-az1245-423"
  }, {
    "name" : "thread",
    "value" : "2340-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.contacts_page_test"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 6,
        "skipped" : 0,
        "passed" : 226,
        "unknown" : 0,
        "total" : 232
      },
      "items" : [ {
        "uid" : "a8b73d02a1cc388f",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/149//#testresult/a8b73d02a1cc388f",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x5587cc55602a <unknown>\n#1 0x5587cc23c5e0 <unknown>\n#2 0x5587cc2347eb <unknown>\n#3 0x5587cc224a09 <unknown>\n#4 0x5587cc226744 <unknown>\n#5 0x5587cc224cbd <unknown>\n#6 0x5587cc22452f <unknown>\n#7 0x5587cc224422 <unknown>\n#8 0x5587cc222493 <unknown>\n#9 0x5587cc222ada <unknown>\n#10 0x5587cc23ef41 <unknown>\n#11 0x5587cc2d08c5 <unknown>\n#12 0x5587cc2b0b22 <unknown>\n#13 0x5587cc2cfd7d <unknown>\n#14 0x5587cc2b08c3 <unknown>\n#15 0x5587cc27e6b3 <unknown>\n#16 0x5587cc27f68e <unknown>\n#17 0x5587cc520a2b <unknown>\n#18 0x5587cc5249b1 <unknown>\n#19 0x5587cc50d225 <unknown>\n#20 0x5587cc525532 <unknown>\n#21 0x5587cc4f238f <unknown>\n#22 0x5587cc544f28 <unknown>\n#23 0x5587cc5450f3 <unknown>\n#24 0x5587cc554e7c <unknown>\n#25 0x7fc761094ac3 <unknown>",
        "time" : {
          "start" : 1728608147608,
          "stop" : 1728608147608,
          "duration" : 0
        }
      }, {
        "uid" : "9cd456cab3921bbe",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/322//#testresult/9cd456cab3921bbe",
        "status" : "passed",
        "time" : {
          "start" : 1728556785950,
          "stop" : 1728556786045,
          "duration" : 95
        }
      }, {
        "uid" : "b2e959b51e6faad8",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/321//#testresult/b2e959b51e6faad8",
        "status" : "passed",
        "time" : {
          "start" : 1728553723619,
          "stop" : 1728553723715,
          "duration" : 96
        }
      }, {
        "uid" : "f0d1449a8e3f6f49",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/148//#testresult/f0d1449a8e3f6f49",
        "status" : "passed",
        "time" : {
          "start" : 1728521811430,
          "stop" : 1728521811521,
          "duration" : 91
        }
      }, {
        "uid" : "ffb799d1dea09b3d",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/320//#testresult/ffb799d1dea09b3d",
        "status" : "passed",
        "time" : {
          "start" : 1728470298344,
          "stop" : 1728470298431,
          "duration" : 87
        }
      }, {
        "uid" : "6b503dda6d9fc272",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/319//#testresult/6b503dda6d9fc272",
        "status" : "passed",
        "time" : {
          "start" : 1728467703778,
          "stop" : 1728467703872,
          "duration" : 94
        }
      }, {
        "uid" : "78ab84fff352117a",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/147//#testresult/78ab84fff352117a",
        "status" : "passed",
        "time" : {
          "start" : 1728435408562,
          "stop" : 1728435408661,
          "duration" : 99
        }
      }, {
        "uid" : "f1bce72eb1238672",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/318//#testresult/f1bce72eb1238672",
        "status" : "passed",
        "time" : {
          "start" : 1728377823349,
          "stop" : 1728377823442,
          "duration" : 93
        }
      }, {
        "uid" : "e025e3a231d7e6f8",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/317//#testresult/e025e3a231d7e6f8",
        "status" : "passed",
        "time" : {
          "start" : 1728375024166,
          "stop" : 1728375024256,
          "duration" : 90
        }
      }, {
        "uid" : "df7cefd37917c13a",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/316//#testresult/df7cefd37917c13a",
        "status" : "passed",
        "time" : {
          "start" : 1728374913476,
          "stop" : 1728374913566,
          "duration" : 90
        }
      }, {
        "uid" : "7df78ed23c1fd0a5",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/146//#testresult/7df78ed23c1fd0a5",
        "status" : "passed",
        "time" : {
          "start" : 1728349002018,
          "stop" : 1728349002110,
          "duration" : 92
        }
      }, {
        "uid" : "50c385ce27327239",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/315//#testresult/50c385ce27327239",
        "status" : "passed",
        "time" : {
          "start" : 1728292504616,
          "stop" : 1728292504707,
          "duration" : 91
        }
      }, {
        "uid" : "c1c39d7301b159c3",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/314//#testresult/c1c39d7301b159c3",
        "status" : "passed",
        "time" : {
          "start" : 1728289545151,
          "stop" : 1728289545241,
          "duration" : 90
        }
      }, {
        "uid" : "15c218fb9975544a",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/145//#testresult/15c218fb9975544a",
        "status" : "passed",
        "time" : {
          "start" : 1728262724028,
          "stop" : 1728262724121,
          "duration" : 93
        }
      }, {
        "uid" : "b0aa8ee1268ddaf6",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/313//#testresult/b0aa8ee1268ddaf6",
        "status" : "passed",
        "time" : {
          "start" : 1728225801032,
          "stop" : 1728225801123,
          "duration" : 91
        }
      }, {
        "uid" : "a28c8b91f851c5f9",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/312//#testresult/a28c8b91f851c5f9",
        "status" : "passed",
        "time" : {
          "start" : 1728220965813,
          "stop" : 1728220965904,
          "duration" : 91
        }
      }, {
        "uid" : "9c7eef0a4b23e80f",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/144//#testresult/9c7eef0a4b23e80f",
        "status" : "passed",
        "time" : {
          "start" : 1728176477444,
          "stop" : 1728176477535,
          "duration" : 91
        }
      }, {
        "uid" : "5907d87d5c18904b",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/311//#testresult/5907d87d5c18904b",
        "status" : "passed",
        "time" : {
          "start" : 1728117149837,
          "stop" : 1728117149923,
          "duration" : 86
        }
      }, {
        "uid" : "4274952f07af6345",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/310//#testresult/4274952f07af6345",
        "status" : "passed",
        "time" : {
          "start" : 1728108562060,
          "stop" : 1728108562151,
          "duration" : 91
        }
      }, {
        "uid" : "59786a5282094a9b",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/143//#testresult/59786a5282094a9b",
        "status" : "passed",
        "time" : {
          "start" : 1728089819989,
          "stop" : 1728089820088,
          "duration" : 99
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "ab6877f419c6b501.json",
  "parameterValues" : [ ]
}